python/skupper_router_internal/management/agent.py

```diff
@@ -506,7 +506,10 @@ def __str__(self):
 
 class HttpListenerEntity(EntityAdapter):
     def create(self):
-        return self._qd.qd_dispatch_configure_http_listener(self._dispatch, self)
+        http_listener = self._qd.qd_dispatch_configure_http_listener(self._dispatch, self)
+        if http_listener is None:
+            raise ValidationError("Invalid httpListener configuration: see logs for details.")
```

Comment:
I was wondering if instead of each entity raising a ValidationError, what if the super class raised a common ValidationError if the create() returned None ?

Reply:
On an unrelated note, I tried the following - 
```
listener {
    host: 0.0.0.0
    port: amqp
    authenticatePeer: no
    saslMechanisms: ANONYMOUS
    sslProfile: test-ssl-profile
}
```
There is no sslProfile called  ```test-ssl-profile``` in my config file.
The router started without issues which does not seem right. Should be bother to fix it ?

Reply:
On a related note, I tried this with this PR - 
```
tcpListener {
    host: 0.0.0.0
    port: 2345
    authenticatePeer: yes
    address: test
    sslProfile: test-ssl-profile
}
```
There is no sslProfile called ```test-ssl-profile``` in my config file and the router started anyway. But with the above config, the router fails to start on main branch. That looks like a regression?

Reply:
I think my next patch for this issue will fix that.

With that patch a config file that is missing the referenced sslProfile will prevent the router from starting.

In the case of skmanage the CREATE of the tcpListener will fail: the sslProfile has to be created first (either in the config file or via skmanage).